### DISCUSSION TITLE
Resources for Continuing to Study R

### DISCUSSION THREAD

**Discussion by cperezcruz** [LiNk-NY/IntroR#49] [General]
Hello!

I wanted to ask how we could go about continuing our studies in R. Is there public coursework elsewhere that we can follow, or other resources to continue practicing on our own? What would you recommend doing after this course?

Thank you :-)

**Comment by ralythompson24** [comment]
Great question, @cperezcruz!

There are a number of resources compiled on our Blackboard site under Weekly Materials > Extra Resources

@LiNk-NY (Marcel, course creator) has also put together this [Learning R](https://trello.com/b/7VsveLu5) Trello board which has some great stuff on it. 

My personal favorite resource for learning R on my own was the [R for Data Science](https://r4ds.hadley.nz/) book, which is an open source textbook created by the chief scientist at Posit Hadley Wickham. In this summer course we introduced you to many of the concepts covered in this book but the writing and examples are really nice and easy to follow if you want to reinforce what you learned over these 5 weeks. 

The Trello board has some nice examples of intermediate to advanced R programming resources as well.

My final note about continuing your R studies - I hope that throughout this course you became comfortable with finding help and resources on coding in R. Mastering your use of R is not about memorizing code or knowing exactly how to approach a problem every time. Having a strong base of the fundamentals in R will really take you a long way, along with having a clear goal and determination to solve whatever problem it is you are facing.

**Comment by LiNk-NY** [comment]
Great question @cperezcruz ! 

I hope you found the Trello board useful!

You may also consider joining community groups like https://rladies.org/ and others https://bookdown.org/manskisa/You_Can_Learn_R/find-your-r-learning-community.html

Best regards,
Marcel
